p2p/module.go

```diff
@@ -28,7 +27,7 @@ const (
 
 type p2pModule struct {
 	bus       modules.Bus
-	p2pConfig modules.P2PConfig // TODO (olshansky): to remove this since it'll be available via the bus
+	p2pConfig *typesP2P.P2PConfig // TODO (olshansky): to remove this since it'll be available via the bus
```

Comment:
In consensus, we renamed `consConfig` to `config` but get `p2pConfig` here. 

My preference is to keep explicit naming for the explicit struct (p2pConfig), but moreso to just be consistent between the two regardless of the approach.